lib/hmmlearn/base.py

```diff
@@ -745,16 +822,12 @@ def _initialize_sufficient_statistics(self):
                  'trans': np.zeros((self.n_components, self.n_components))}
         return stats
 
-    def _accumulate_sufficient_statistics(self, stats, X, lattice,
+    def _accumulate_sufficient_statistics(self, X, lattice,
```

Comment:
I think the change in _accumulate_sufficient_statistics should occur first, and in a separate PR, so that that API can be ironed out.  It will also need some changelog entries so that anyone who implemented custom HMM subclasses knows what they need to change.